base/pmap.jl

```diff
@@ -64,9 +58,13 @@ Split a collection into at least `min_batch_count` batches.
 Equivalent to `partition(c, max_batch_size)` when `length(c) >> max_batch_size`.
 """
 function batchsplit(c; min_batch_count=1, max_batch_size=100)
+    if min_batch_count < 1
+        throw(ArgumentError("min_batch_count must be > 0, got $min_batch_count"))
+    end
 
-    @assert min_batch_count > 0
-    @assert max_batch_size > 1
+    if max_batch_size < 1
```

Comment:
As long as 1 works correctly, I think it must be allowed. While currently it is not used anywhere else, in general, the values may be runtime configurable in a user program - for example, a config file or the environment. In such cases it is better to support all correctly computable values.